doofinder.php

```diff
@@ -2445,135 +2445,114 @@ public function autoinstaller($apikey, $api_endpoint, $admin_endpoint)
         //Require only on this function to not overload memory with not needed classes
         require_once _PS_MODULE_DIR_ . 'doofinder/lib/EasyREST.php';
         $client = new EasyREST();
-        $dfhost = 'https://'.$api_endpoint;
         $api_endpoint_array = explode('-', $api_endpoint);
         $region = $api_endpoint_array[0];
         $shops = Shop::getShops();
         $this->manualInstallation();
         foreach ($shops as $shop) {
             $languages = Language::getLanguages(true, $shop['id_shop']);
+            $primary_lang_id = Configuration::get('PS_LANG_DEFAULT');
+            $primary_language = reset(array_filter($languages, function ($lang) use ($primary_lang_id) {
+                return (int)$lang["id_lang"] === (int)$primary_lang_id;
+            }));
+            $primary_language_iso_code = $primary_language["iso_code"];
+
```

Comment:
```suggestion
            $primary_lang_id = Configuration::get('PS_LANG_DEFAULT', null, $shop['id_shop_group'], $shop['id_shop']);
            $primary_language_iso_code = Language::getIsoById($primary_lang_id)

```
- Add the id_shop when obtaining the language, so that it gives you the default language of the store that you indicate.
- Function Language::getIsoById gives the ISO of the language.

Reply:
That's great, thanks